CMakeLists.txt

```diff
@@ -35,7 +43,7 @@ install(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION lib ARCHIVE DESTINATION lib
 
 ###################################################################################
 # Build Pybind wrapper
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/wrap/cmake")
+find_package(gtwrap)
```

Comment:
I couldn't find where gtwrap-config.cmake or FindGtWrap.cmake are defined?

From the removed line above, I expected it to be in the wrap repo, but couldn't find it. Sorry if it's a silly question, but I never built the python wrapper for gtsam so I'm unfamiliar with it.

Please, add a comment at this point explaining what is gtwrap and where it comes from.

Reply:
That's my bad. You'll have to install the PR branch version of wrap as detailed here: https://github.com/borglab/wrap/pull/11

It should have all the instructions you need to install the CMake modules.